subprojects/emcee/queue-api/src/main/kotlin/com/avito/emcee/queue/TestExecutionBehavior.kt

```diff
@@ -0,0 +1,9 @@
+package com.avito.emcee.queue
+
+import com.squareup.moshi.Json
+
+public data class TestExecutionBehavior(
+    val environment: Map<String, String>,
+    @Json(name = "numberOfRetries")
+    val retries: Int
```

Comment:
There are some new fields out there
https://github.com/avito-tech/Emcee/blob/master/Sources/RunnerModels/TestExecutionBehavior.swift

If you don't need them, I guess you just don't parse them.